content/_index.md

```diff
@@ -67,8 +67,8 @@ along the way.
   just between your own devices.
 
 - **Portable.** Configure and monitor Syncthing via a responsive and powerful
-  interface accessible via your browser. Works on Mac OS X, Windows, Linux,
-  FreeBSD, Solaris and OpenBSD. Run it on your desktop computers and synchronize
+  interface accessible via your browser. Works on macOS, Windows, Linux, Android,
+  FreeBSD, Solaris and OpenBSD. Run it on your desktop computers, phones and tablets, and synchronize
```

Comment:
Perhaps reorder by actual usage stats and simplify the (now even longer) list of random unixes?

```suggestion
  interface accessible via your browser. Works on Linux, Windows, Android, macOS, 
  and other Unixes. Run it on your desktop computers, phones and tablets, and synchronize
```